pages/design/composed-template-empty.md

```diff
@@ -0,0 +1,78 @@
+---
+name:
+description: |
+  This rule checks ...
+
+success_criterion:
+- x.y.z
+
+atomic_rules:
+- 
+
+authors:
+-
+---
+
+## Aggregation Definition
+
+### Applicability
+
+The rule applies to any (??) element ...
+
+### Expectation
+
+For each test target, the outcome of << Logic here >>:
+
+- 
+- 
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- 
+
+
+## Test Cases
+
+### Pass example 1
```

Comment:
Use "Passed", "Failed", "Inapplicable" as defined in https://www.w3.org/TR/act-rules-format/#output-outcome